packages/editor/src/editor/canvas/data-table-column/DataTableColumnDialog.tsx

```diff
@@ -0,0 +1,154 @@
+import {
+  Button,
+  Dialog,
+  DialogClose,
+  DialogContent,
+  DialogFooter,
+  DialogHeader,
+  DialogTitle,
+  DialogTrigger,
+  SelectRow,
+  Table,
+  TableBody,
+  TableCell,
+  useTableSelect,
+  type BrowserNode
+} from '@axonivy/ui-components';
+import { IvyIcons } from '@axonivy/ui-icons';
+import { flexRender, type ColumnDef, useReactTable, getCoreRowModel, getFilteredRowModel } from '@tanstack/react-table';
+import { useMeta } from '../../../context/useMeta';
+import { useAppContext } from '../../../context/AppContext';
+import { useEffect, useState } from 'react';
+import type { Variable } from '@axonivy/form-editor-protocol';
+import { findAttributesOfType, findVariablesOfType } from '../../../data/variable-tree-data';
+import { createInitiTableColumns, useData } from '../../../data/data';
+import { componentByName } from '../../../components/components';
+
+export const DataTableColumnDialog = ({ id, initValue }: { id: string; initValue: string }) => {
```

Comment:
If you keep it simple that a user needs to define this value via the property view then this whole thing is not needed...

Reply:
at least the browser not, and the other thing is only a button which creates the columns